api/queues/BuildTasksQueue.js

```diff
@@ -0,0 +1,24 @@
+const { Queue } = require('bullmq');
+
+const BuildTasksQueueName = 'build-tasks';
+
+class BuildTasksQueue extends Queue {
+  constructor(connection) {
+    super(BuildTasksQueueName, {
+      connection,
+      defaultJobOptions: {
+        attempts: 3,
+        backoff: {
+          type: 'exponential',
+          delay: 3000,
+        },
+        delay: 2 * 60 * 1000, // 2 minutes
```

Comment:
What are the advantages of adding a default `delay` on job kickoff?

Reply:
Oversight, I think I copied this config over from another queue but I'll edit